tslint.json

```diff
@@ -18,7 +18,6 @@
     "interface-name": false,
     "jsdoc-format": true,
     "label-position": true,
-    "label-undefined": true,
```

Comment:
@DxCx the tslint changes weren't made by me. Is an already merged pull request into master :(